content/en/developers/dogstatsd/unix_socket.md

```diff
@@ -103,6 +103,24 @@ To enable the Agent DogStatsD UDS:
 
         **Note**: Remove `readOnly: true` if your application containers need write access to the socket.
 
+**Note**: EKS Fargate doesn't support mounting host paths. Alternatively, the socket can be shared between the agent sidecar container and the application container using a `emptyDir` volume.
+    - Mount the socket folder in your `datadog-agent` container and the application container in the pod:
```

Comment:
you might split it in two items

* Create the volume  (emptyDir) in the applicative pod.
* Mount the volume in each pod's containers